async-http-client-backend/fs2/src/main/scala/com/softwaremill/sttp/asynchttpclient/fs2/AsyncHttpClientFs2Backend.scala

```diff
@@ -35,25 +35,23 @@ class AsyncHttpClientFs2Backend[F[_]: ConcurrentEffect] private (asyncHttpClient
       .compile
       .fold(ByteBuffer.allocate(0))(concatByteBuffers)
 
-    implicitly[Effect[F]].map(bytes)(_.array())
+    Functor[F].map(bytes)(_.array())
   }
 }
 
 object AsyncHttpClientFs2Backend {
 
-  private def apply[F[_]: ConcurrentEffect](asyncHttpClient: AsyncHttpClient, closeClient: Boolean)(
-      implicit t: Timer[F]): SttpBackend[F, Stream[F, ByteBuffer]] =
+  private def apply[F[_]: ConcurrentEffect](asyncHttpClient: AsyncHttpClient,
+                                            closeClient: Boolean): SttpBackend[F, Stream[F, ByteBuffer]] =
     new FollowRedirectsBackend(new AsyncHttpClientFs2Backend(asyncHttpClient, closeClient))
 
-  def apply[F[_]: ConcurrentEffect](options: SttpBackendOptions = SttpBackendOptions.Default)(
-      implicit t: Timer[F]): SttpBackend[F, Stream[F, ByteBuffer]] =
-    AsyncHttpClientFs2Backend[F](AsyncHttpClientBackend.defaultClient(options), closeClient = true)
+  def apply[F[_]: ConcurrentEffect](
+      options: SttpBackendOptions = SttpBackendOptions.Default): SttpBackend[F, Stream[F, ByteBuffer]] =
+    apply[F](AsyncHttpClientBackend.defaultClient(options), closeClient = true)
```

Comment:
Note: I changed to `apply` for discoverability of the first (private) `apply`'s usages in IDEA.